styles.css

```diff
@@ -271,7 +271,7 @@ button:active {
 }
 
 .card:hover {
-  background-color: #60c095;
+  background-color: #227150;
```

Comment:
Here, we added a darker green tone to increase contrast for the website's readability.